contracts/clients/src/Aggregator.ts

```diff
@@ -38,12 +39,18 @@ export type BundleReceiptError = {
   submitError: string | undefined;
 };
 
-export type BundleReceipt = {
-  transactionIndex: number;
-  transactionHash: string;
+/**
+ * The BLS Wallet specific values in a {@link BundleReceipt}.
+ */
+export type BlsWallet = {
   bundleHash: string;
-  blockHash: string;
-  blockNumber: number;
+};
+
+/**
+ * The bundle receipt returned from a BLS Wallet Aggregator instance. It is a combination of an ethers {@link ContractReceipt} and a {@link BlsWallet} type.
+ */
+export type BundleReceipt = ContractReceipt & {
```

Comment:
Will this updated type break anything in the extension?

Reply:
Tested the extension locally and is able to get the transaction receipt successfully. The transaction hashes lined up with what was logged in the aggregator logs.

Thinking about it, I might check it works on testnet as well as locally - then I can check the transaction hash works on etherscan

Reply:
Just tested successfully via quill running a local aggregator pointing at arbitrum goerli.

The extension does not query the hash via `bundleReceipt.bundleHash`, so it still works (if it did query the hash this way, it would break).

It appears to [save the bundle hash here](https://github.com/web3well/bls-wallet/blob/b6fc2164a3a6b8785c88f18079e38b80fd7caeba/extension/source/background/AggregatorController.ts#L91-L94) so it doesn't have to query it.

It also uses the [returned bundle hash here](https://github.com/web3well/bls-wallet/blob/b6fc2164a3a6b8785c88f18079e38b80fd7caeba/extension/source/Home/Wallet/Wallets/SendDetail/SendDetail.tsx#L111) to get the transaction receipt.

Can't see anywhere else it could break to my knowledge.